README.md

```diff
@@ -25,23 +25,53 @@ Currently tested platforms:
 
 #### On a Linux machine make sure to add your computer user to the ``docker`` group
 
+If your user is not added in the group, it causes a permission error when starting Docker.
+
+The instruction is [here](https://docs.docker.com/engine/install/linux-postinstall/).
+
+#### Make sure `docker swarm` works
+
+If you find the following error, please refer [this instruction](https://forums.docker.com/t/28428/8) to remove `--live-restore` option.
+
+```
+$ docker swarm init
+Error response from daemon: --live-restore daemon configuration is incompatible with swarm mode
+```
+
+Then make sure `docker swarm` works.
+
+```
+docker swarm init
+docker swarm leave --force
+```
+
+#### Make sure to enable cgroups v1.
+
+If you find the following error, plesae enable cgroups v1. The instraction is [here](https://fedoramagazine.org/docker-and-fedora-32/).
```

Comment:
The fine line between thorough and excessive documentation is hard to find.

Our software relies on Docker, is it reasonable to assume that it is the user's responsibility to have a 100% working installation of Docker before proceeding to installation of our repo?

This might be better as a wiki page linked to by README.md: "before starting, make sure Docker works. --> wiki link"

The bottom line is that **Fedora 32 does not support docker** [1] [2]. That is a huge problem. The most direct solution is obviously to use a supported platform (Ubuntu 20.04, any other Linux, Docker on Mac, a VM, or possibly Fedora 31).
Fedora 32 users are responsible for figuring out their own Docker problems as they are platform-specific.

[1] https://docs.docker.com/engine/install/fedora/

> To install Docker Engine, you need the 64-bit version of one of these Fedora versions:
> 
> - Fedora 30
> - Fedora 31

[2] https://fedoramagazine.org/docker-and-fedora-32/

> At the time of writing, Docker is not supported on Fedora 32.

Reply:
> Our software relies on Docker, is it reasonable to assume that it is the user's responsibility to have a 100% working installation of Docker before proceeding to installation of our repo?
> 
> This might be better as a wiki page linked to by README.md: "before starting, make sure Docker works. --> wiki link"

I agree. It's better to move the precondition part to run docker to the wiki page.

Reply:
> The most direct solution is obviously to use a supported platform (Ubuntu 20.04, any other Linux, Docker on Mac, a VM, or possibly Fedora 31).

Yeah, Fedora 32 users need to fix it by themselves in this case. I am happy to prepare the wiki page. But as I do not work for this project every day, if you like, you can go a head.

```
$ cat /etc/fedora-release 
Fedora release 32 (Thirty Two)

$ docker run --rm -t hello-world
...
Hello from Docker!
This message shows that your installation appears to be working correctly.
...
```

Reply:
Are you using podman or moby?

The SELinux troubles only happen when mounting volumes, stateless containers have fewer problems.

Reply:
I am using moby. The `moby-engine` RPM package in Fedora repository.

```
$ rpm -qf /usr/bin/docker
moby-engine-19.03.11-1.ce.git42e35e6.fc32.x86_64
```

> The SELinux troubles only happen when mounting volumes, stateless containers have fewer problems.

That's good information to check. Thanks.
SELinux is enabled right now.

```
$ sestatus 
SELinux status:                 enabled
SELinuxfs mount:                /sys/fs/selinux
SELinux root directory:         /etc/selinux
Loaded policy name:             targeted
Current mode:                   enforcing
Mode from config file:          enforcing
Policy MLS status:              enabled
Policy deny_unknown status:     allowed
Memory protection checking:     actual (secure)
Max kernel policy version:      32
```